### DISCUSSION TITLE
Does Github have a digital legacy for developers?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22547] [New to GitHub]
<h4>
<a name="about-1" class="anchor" href="#about-1"></a>about</h4>
<p>Apple ID <strong>Digital Legacy</strong> is a feature in iOS 15 that allows you to select friends, family or people you trust to securely access your data such as photos, videos and more in the event of death. To prevent your important data from being simply lost as a last resort.</p>
<h4>
<a name="why-is-it-important-2" class="anchor" href="#why-is-it-important-2"></a>Why is it important?</h4>
<ol>
<li>I don’t know if I’m going to live long, I’d like to configure this so that some developers have access to my repository in case I die.</li>
<li>Many companies like Facebook, Google from what I read and I know have implemented this digital legacy feature, as well as Apple. Do you have this goal of providing a digital legacy for developers?</li>
<li>I didn’t find anything talking about it, if you have any information I would appreciate it.</li>
</ol>
<h4>
<a name="notes-3" class="anchor" href="#notes-3"></a>Notes</h4>
<p>I hope someone reads and can clarify this doubt.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>how about this at your Settings?</p>
<p><a href="https://github.com/settings/admin">https://github.com/settings/admin</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/0/70a5297111a16bf0bf33a12d59162e9fafa04bbb.png" data-download-href="/uploads/short-url/g4vkJoo162w46LFvqFPJzHKdlRV.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071758-5674af6f-7b61-4987-a203-43337de1f912.png" alt="image" data-base62-sha1="g4vkJoo162w46LFvqFPJzHKdlRV" width="690" height="306" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/0/70a5297111a16bf0bf33a12d59162e9fafa04bbb_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">847×376 18.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ghost** [comment]
<p>you are wonderful - thank you very much for the feedback, they are options that we get lost. I didn’t notice it had this feature.</p>
